libs/openFrameworksCompiled/project/makefileCommon/config.linux.common.mk

```diff
@@ -142,10 +142,10 @@ ifeq ($(CXX),g++)
 	endif
 else
 	ifeq ($(CXX),g++-5)
-		PLATFORM_CFLAGS = -Wall -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0
+		PLATFORM_CFLAGS = -Wall -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -DGCC_HAS_REGEX
 	else
 	    ifeq ($(CXX),g++-4.9)
-		    PLATFORM_CFLAGS = -Wall -std=c++14
+		    PLATFORM_CFLAGS = -Wall -std=c++14 -DGCC_HAS_REGEX
```

Comment:
I've added the defines to these two checks, assuming gcc 5 must support `std::regex`, too.